OpenGL/Constructs/VAO.cs

```diff
@@ -651,6 +668,26 @@ private void DrawInstancedOGL2(int count)
             throw new Exception("The Function glDrawElementsInstanced was first added in OpenGL 3.1 which is not supported on this system.");
         }
 
+        /// <summary>
+        /// OGL4 method uses a vertex array object for quickly binding the VBOs to their attributes.
+        /// </summary>
+        private void MultiDrawElementsIndirectOGL43(VBO<DrawElementsIndirectCommand> cmdVBO, int cmdCount)
+        {
+            if (ID == 0) return;
+            Gl.BindVertexArray(ID);
+            Gl.BindBuffer(cmdVBO);
+            Gl.MultiDrawElementsIndirect(DrawMode, elementType, IntPtr.Zero, cmdCount, 0);
+            Gl.BindVertexArray(0);
+        }
+
+        /// <summary>
+        /// OGL version does not support Gl.MultiDrawElementsIndirect.
+        /// </summary>
+        private void MultiDrawElementsIndirectOGLTooOld(VBO<DrawElementsIndirectCommand> cmdVBO, int cmdCount)
+        {
+            throw new Exception("The Function glMultiDrawElementsIndirect was first added in OpenGL 4.3 which is not supported on this system.");
```

Comment:
Would it be relatively easy to emulate this command?  I took a look at the docs, and it looks like we would need to read back data from the GPU (or have cached it locally) to make this work, so it could be a non-starter.